src/app/rundown-view/components/rundown-tooltips/mini-shelf-tooltip/mini-shelf-tooltip.component.html

```diff
@@ -0,0 +1,6 @@
+<div #tooltipHoverElement (mouseenter)="emitNewHoverMouseEvent($event)" (mousemove)="emitNewHoverMouseEvent($event)" (mouseleave)="emitNewHoverMouseEvent()" class="c-hover-scrub-hover-element"></div>
+
+<sofie-tooltip *ngIf="tooltipHoverElement" [tooltipElementHoverMouseEventObservable]="mouseHoverEventSubject.asObservable()">
```

Comment:
This `*ngIf` is always true and should therefore be deleted.

Reply:
Done